src/components/AboutUs/team.json

```diff
@@ -0,0 +1,345 @@
+    "title": "Web Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/dania-deyar-097558243/",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/dania-diar",
+        "icon": "github"
+      }
+    ]
+  },
+ 
+  {
+    "id": "5",
+    "name": "Rondik Mousa",
+    "img": "https://ca.slack-edge.com/T03K5TU7K0E-U03ME7K3Q07-7c77b4b5ec30-512",
+    "title": "Web Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/rondik-mousa-646653256/",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/rondikmousa",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "6",
+    "name": "Helin Faidhan",
+    "img": "https://ca.slack-edge.com/T03K5TU7K0E-U03NTLZN3MW-31f24e5340ee-512",
+    "title": "Web Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/helin-faidhan-a516a71aa/",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/Helin-Faidhan",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "7",
+    "name": "Marwa Hakmi",
+    "img": "https://ca.slack-edge.com/T03K5TU7K0E-U03M7GU31T9-7f79896abdee-512",
+    "title": "Web Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https:www.linkedin.com/in/marwa-hakmi",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/MARWAHAKMI",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "8",
+    "name": "Ayub Abdullah",
+    "img": "https://ca.slack-edge.com/T03K5TU7K0E-U03MFBY877G-fa68f334cdc6-512",
+    "title": "Web Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/mwlite/in/ayub-abdullah",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/ayubabdullah",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "9",
+    "name": "Omar Bassatni",
+    "img": "https://ca.slack-edge.com/T03K5TU7K0E-U03ML8M4TDK-39b1e18f3beb-512",
+    "title": "Web Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/omar-bassatni-40a762188/",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/OmarBassatni97",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "10",
+    "name": "Sarah Abou Dalal",
+    "img": "https://media-exp1.licdn.com/dms/image/D4E03AQGlv3KsvBy9mA/profile-displayphoto-shrink_400_400/0/1667645813651?e=1673481600&v=beta&t=cUf6GPWuvmr0KWx1aBIDtcSpGvb3rAS0lGRcIrzOX10",
```

Comment:
for some reason my image has a black bg, imma send u another one i think it will be fixed